### PR TITLE
Allow adding filmic auto-tune to quick access panel

### LINKED ISSUE #12665: quick access auto tune levels filmic rgb
**Is your feature request related to a problem? Please describe.**
In shortcuts, it is possible to set a shortcut for filmic rgb auto tune levels. However, it is not possible to add auto tune levels to the quick access menu.

**Describe the solution you'd like**
Ability to add filmic rgb auto tune levels in quick access menu.

**Additional context**
dt 4.0.1
mint cinnamon 21

### PR DESCRIPTION
**Pull request by Nilvus**
Fix #12665. For that, this need to revert a work I've done on #11372 but it's more important to let users had filmic auto-tune line on quick access panel than having hover effect on the picker (we don't have it on sliders ones). So revert back.

But, @dterrahe I need your help. I don't understand how to let shortcut work now. I can set it with that PR but not use it. Could you post in a comment the correct code. My coding skills remains really limited.

This PR also improve line between module (CSS code). This follows a pixls.us comment: https://discuss.pixls.us/t/discussion-around-new-filters-widgets/32490/26?u=nilvus

### CODE REVIEW

File: src/iop/filmicrgb.c [hunk 1]
```diff
@@ -4248,10 +4248,8 @@ void gui_init(dt_iop_module_t *self)
                                                     "useful to give a safety margin to extreme luminances."));
 
   // Auto tune slider
-  GtkWidget *hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 0);
-  gtk_box_pack_start(GTK_BOX(hbox), dt_ui_label_new(_("auto tune levels")), TRUE, TRUE, 0);
-  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, NULL);
-  gtk_box_pack_start(GTK_BOX(hbox), g->auto_button, FALSE, FALSE, 0);
+  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, dt_bauhaus_combobox_new(self));
+  dt_bauhaus_widget_set_label(g->auto_button, NULL, N_("auto tune levels"));
   dt_action_define_iop(self, NULL, N_("auto tune levels"), GTK_WIDGET(g->auto_button), &dt_action_def_toggle);
```
Comment (dterrahe) [inline src/iop/filmicrgb.c:4253]: Please also remove this `dt_action_define_iop`; the shortcut is already fully set up by `dt_bauhaus_widget_set_label`.

Normally, buttons "attached to" a combo need to be called with a shortcut that has _element_ "button". But in this case it makes sense to not require that, for easier, more sensible, setup, but also to not break shortcuts for people that now have them set up via the button.

Please change
https://github.com/darktable-org/darktable/blob/26c52b36b5e5371488b9ec9ed7d156afad5077f1/src/bauhaus/bauhaus.c#L3420
and
https://github.com/darktable-org/darktable/blob/26c52b36b5e5371488b9ec9ed7d156afad5077f1/src/bauhaus/bauhaus.c#L3456
to
`if(element == DT_ACTION_ELEMENT_BUTTON || !w->data.combobox.entries->len)`

Comment (Nilvus) [reply src/iop/filmicrgb.c]: Thanks. Should be fixed on last commit.

Comment (dterrahe) [reply src/iop/filmicrgb.c]: Haven't tested, but looks good.

**Review by dterrahe — APPROVED** [review]

**Review by TurboGit — APPROVED** [review]
Thanks! Works for me too.

File: src/iop/filmicrgb.c [hunk 2]
```diff
@@ -4248,19 +4248,16 @@ void gui_init(dt_iop_module_t *self)
...
 
   // Auto tune slider
-  GtkWidget *hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 0);
-  gtk_box_pack_start(GTK_BOX(hbox), dt_ui_label_new(_("auto tune levels")), TRUE, TRUE, 0);
-  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, NULL);
-  gtk_box_pack_start(GTK_BOX(hbox), g->auto_button, FALSE, FALSE, 0);
-  dt_action_define_iop(self, NULL, N_("auto tune levels"), GTK_WIDGET(g->auto_button), &dt_action_def_toggle);
+  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, dt_bauhaus_combobox_new(self));
+  dt_bauhaus_widget_set_label(g->auto_button, NULL, N_("auto tune levels"));
   dt_gui_add_class(g->auto_button, "dt_bauhaus_alignment");
```
Comment (dterrahe) [inline src/iop/filmicrgb.c:4253]: looks like this line needs to go as well.

Comment (Nilvus) [reply src/iop/filmicrgb.c:4253]: indeed. Will remove that in a future PR. Thanks for noticing it.

**Pull request merged**